src/Kernel/ready.js

```diff
@@ -1,14 +1,10 @@
-import { ipcRenderer } from 'electron'
-
-import * as Web3 from '../Actions/Web3'
 import * as Core from '../Actions/Core'
-import * as Accounts from '../Actions/Accounts'
-import * as Logs from '../Actions/Logs'
-import * as Config from '../Actions/Config'
-
-import * as ErrorHandler from './ErrorHandler'
 
 import { initAutoUpdates } from '../Init/Renderer/AutoUpdate'
+import { initCore } from '../Init/Renderer/Core'
```

Comment:
Since `ready.js` is the renderer initialization entrypoint (unless I'm mistaken) perhaps it should be moved to `src/Init/Renderer/index.js`?

Ideally the `Kernel` directory wouldn't have continued to exist after this commit.